web/client/utils/TemplateUtils.js

```diff
@@ -33,7 +33,16 @@ export const validateStringAttribute = (feature, attribute, start = 0, end = 0)
 export const getCleanTemplate = (chosenTemplate, feature, regex, start = 0, end = 0, getDefaultMissingProperty = () => "") => {
     const matchVariables = isString(chosenTemplate) && chosenTemplate.match(regex);
     const replacedTag = matchVariables && matchVariables.map(temp => {
-        const varReplaced = temp.replace(/(<([^>]+)>)/ig, '');
+        let varReplaced = temp.replace(/(<([^>]+)>)/ig, '');
+        if (varReplaced.indexOf("-") !== -1) {
```

Comment:
I should not do this, but improve error messages and add a suggestion about the correct syntax.